view/live/live.js

```diff
@@ -638,7 +638,8 @@ steal('can/util',
 				can.data(wrapped, 'hooks', hooks = {});
 			}
 			// Get the attribute value.
-			var attr = elements.getAttr(el, attributeName),
+			// Cast to String. String expected for rendering. Attr may return other types for some attributes.
+			var attr = String(elements.getAttr(el, attributeName)),
```

Comment:
Should we changes this to an implicit cast like `'' + elements.getAttr(el, attributeName)`